.github/CONTRIBUTING.md

```diff
@@ -18,11 +18,34 @@ See our guide on [how to create a great issue](https://code-review.tidyverse.org
 
 ### Pull request process
 
-*   Fork the package and clone onto your computer. If you haven't done this before, we recommend using `usethis::create_from_github("JRaviLab/MolEvolvR", fork = TRUE)`.
+*   Fork the package and clone onto your computer. If you haven't done this before, we recommend using `usethis`.
 
-*   Install all development dependencies with `devtools::install_dev_deps()`, and then make sure the package passes R CMD check by running `devtools::check()`. 
-    If R CMD check doesn't pass cleanly, it's a good idea to ask for help before continuing. 
-*   Create a Git branch for your pull request (PR). We recommend using `usethis::pr_init("brief-description-of-change")`.
+*   Install and load the `usethis` package with:
+    ```
+    install.packages("usethis")
+    
+    library("usethis")
+    ```
+*   Clone and fork the MolEvolvR package using:
+    ```
+    usethis::create_from_github("JRaviLab/MolEvolvR", fork = TRUE)
+    ```
+*   Install all development dependencies and then make sure the package passes R CMD check using `devtools`:
```

Comment:
As this package includes Bioconductor dependencies, we should probably include instructions on installing [BiocManager](https://www.bioconductor.org/install/) as well. Perhaps after this section, but before `devtools::install_dev_deps()`?